core/wallet/transaction.go

```diff
@@ -175,10 +179,10 @@ func ConfirmDepositProcess(ctx context.Context, n *core.IpfsNode, prepareRespons
 			for i := 0; i < 10; i++ {
 				err = grpc.EscrowClient(escrowService).WithContext(ctx,
 					func(ctx context.Context, client escrowPb.EscrowServiceClient) error {
-						_, err = client.CloseChannel(ctx, signSuccessChannelState)
-						if err != nil {
-							return err
-						}
+						//_, err = client.CloseChannel(ctx, signSuccessChannelState)
+						//if err != nil {
+						//	return err
+						//}
```

Comment:
Comment out entire loop? Nothing to call right now.

Reply:
ah, it was a mistake. it was my purpose of commenting out them to mock a test case and i forgot to undo...